manager/integration/tests/test_zone.py

```diff
@@ -14,8 +14,14 @@
 from random import randrange
 from test_scheduling import wait_new_replica_ready
 from common import get_version_api_client
+from common import CONDITION_STATUS_TRUE
+from common import wait_for_volume_condition_scheduled
+from common import wait_for_volume_delete
+from test_scheduling import SETTING_REPLICA_SOFT_ANTI_AFFINITY
 
 
+SETTING_ZONE_SOFT_ANTI_AFFINITY = "replica-zone-soft-anti-affinity"
+
 # label deprecated for k8s >= v1.17
 DEPRECATED_K8S_ZONE_LABEL = "failure-domain.beta.kubernetes.io/zone"
```

Comment:
I think for k8s >= v1.17, the label is `topology.kubernetes.io/zone`, and for k8s <v1.17, it's `failure-domain.beta.kubernetes.io/zone`.

Reply:
Yes, you're totally right, we've this function to return the correct label based on kubernetes version

https://github.com/longhorn/longhorn-tests/blob/f70218fad05fafc1f3fc59f144f631b44b18d6f3/manager/integration/tests/test_zone.py#L34